Application/components/Server/MemberList.tsx

```diff
@@ -173,14 +172,16 @@ export function MemberList({isAdmin, chatID}: any) {
           console.error('Error fetching member list:', error);
         }
       };
-
+      createErrorMessage();
+      setChannelKey(chatID)
+      console.log("I am the key "+ channelKey)
```

Comment:
All `console.log` statements should be deleted in the future once a feature has been integrated